compliance_checker/base.py

```diff
@@ -549,7 +549,7 @@ def attr_check(kvp, ds, priority, ret_val, gname=None, var_name=None):
     # starting with "check".  Avoid naming check functions
     # starting with check if you want to pass them in with
     # a tuple to avoid them being checked more than once
-    elif hasattr(other, "__call__"):
+    elif callable(other):
```

Comment:
I wrote this, didn't even realize that this was preferred style...

Reply:
IMO what you had there is "better style" but it can fail in some corner cases. The callable check won't.